components/resources/BUILD.gn

```diff
@@ -76,6 +77,15 @@ repack("resources") {
     ]
   }
 
+  if (enable_playlist) {
```

Comment:
this should live in `components/playlist/resources/BUILD.gn`

Reply:
We did similar thing for speedreader, AFAIK this is the preferred method https://github.com/brave/brave-core/blob/master/components/speedreader/resources/BUILD.gn#L22

Reply:
@iefremov 
I think any pak file should be included in one of toplevel pak files (brave_XXX_percent.pak or brave_resources.pak) finally - https://github.com/brave/brave-core/blob/master/brave_paks.gni#L195 because they are bundled at last.

W/o this change, playlist pak file is not included any toplevel pak files.
You can see many other pak files are added in this file's sources.
Or directly added to brave_resources.pak target like https://github.com/brave/brave-core/blob/master/brave_paks.gni#L89

and I think speedreader_resources.pak is not generated and not bundled now. Can you check?

Reply:
Huh, good catch! Actually I moved speedreader resources to `components/resources/speedreader_resources.grdp` and forgot to cleanup the pak file. So now I'm a bit confused about using `.pak` vs `.grdp`